utils/osgenerator/pb/mockos.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package pb;
+
+message MockOS {
+    string version = 1;
+    string cookie = 2;
+    bytes data  = 3;
+    bytes hash = 4;
+    bool supported = 5;
```

Comment:
Add a comment stating that this flag is used to say that the OS image is valid but not supported by the Target.